ntoskrnl/kdbg/i386/i386-dis.c

```diff
@@ -3128,32 +3128,32 @@ append_seg (void)
   if (prefixes & PREFIX_CS)
     {
       used_prefixes |= PREFIX_CS;
-      oappend ("%cs:" + intel_syntax);
+      oappend (&"%cs:"[intel_syntax]);
```

Comment:
Is that some special GCC style sauce?

Reply:
No, it's just array offset syntax, equivalent to the addition. To clang this makes it obvious that the code isn't trying to do string concat here.